proteus/MeshTools.py

```diff
@@ -6338,9 +6338,10 @@ def setOutputFiles(self, name='mesh', poly=True, ply=False, asymptote=False, geo
         self.outputFiles['geo'] = gmsh
 
 
-def msh2triangle(fileprefix):
+def msh2triangle(fileprefix, nd):
```

Comment:
Since this function can convert a .msh file into Tetgen readable files, I think the function name should be modified to reflect this capability (msh2simplex?).